testutils/goroutines/verify.go

```diff
@@ -77,16 +77,32 @@ func filterStacks(stacks []Stack, opts *VerifyOpts) []Stack {
 	return filtered
 }
 
+var retryStates = map[string]struct{}{
+	"runnable": struct{}{},
+	"running":  struct{}{},
+	"syscall":  struct{}{},
+}
+
+func shouldRetry(stack Stack) bool {
+	if _, ok := retryStates[stack.State()]; ok {
+		return true
+	}
+
+	if stack.State() == "sleep" {
+		// Go 1.5 has a time.Sleep in the GC path. If we find the exact
+		// time.Sleep call, we should treat this as a runnable goroutine.
+		// see: https://github.com/uber/tchannel-go/issues/151
+		return bytes.Contains(stack.Full(), []byte("time.Sleep(0x186a0)"))
```

Comment:
take a look at the issue for more info, it ends up being caused by:
https://github.com/golang/go/blob/release-branch.go1.5/src/runtime/mgcmark.go#L304

however, the stack trace shows all values as hex, so the 100 microseconds ends up looking like `0x186a0`.

(Luckily, go 1.6 removed this path completely!)